interaction-manager/src/lib_components/dropzone.js

```diff
@@ -254,6 +255,18 @@ class Dropzone extends DragBase {
     handleDrop() {
         let dropType = this.options.dropType;
 
+        const eventData = {
+            preventDefault: () => {
+                dropType = 'ignore';
+            },
+            target: this.draggedElement,
+            dropzone: this.draggedOver,
+        };
+
+        this.options.onDrop && this.options.onDrop(eventData);

```

Comment:
Maybe we need to document somewhere that you should not have async code in the onDrop event that calls `event.preventDefault` otherwise it will not prevent dropping the element.